docs/indexer-schema-v2.md

```diff
@@ -52,10 +52,7 @@ erDiagram
         Int protocolVersion "required"
         Int activeBinId "optional, V2 only"
         Int binStep "optional, V2 only"
-        Float basePrice "optional, V2 only"
         BigInt globalLiquidity "optional, V2 only"
-        Float baseFee "optional, V2 only"
```

Comment:
We can either remove that field and frontend will get this data, or we have to add this field to PoolCreatedEvent, which is not perfect given the audit is now done.